### ISSUE TITLE
Not able to integrate with angular application

### ISSUE THREAD

**Issue by devraj112** [plurimath/plurimath-js#31]
While installing the plurimath, following the installation method provided in the readme file:
```
Node.JS
$ npm install -S @plurimath/plurimath
ES Modules (recommended):

import Plurimath from "@plurimath/plurimath"

const formula = new Plurimath('ubrace(1+2+3+4)_("4 terms")', "asciimath");
console.log(formula.toLatex());
```
After that I encountered this TypeScript errors related to missing namespaces `Opal`.
```
Error: node_modules/@plurimath/plurimath/dist/index.d.ts:3:11 - error TS2503: Cannot find namespace 'Opal'.

3     data: Opal.Plurimath.Math.ParserResult;
            ~~~~

Error: node_modules/@plurimath/plurimath/dist/index.d.ts:4:39 - error TS2503: Cannot find namespace 'Opal'.

4     constructor(data: string, format: Opal.Plurimath.Math.Format);
                                        ~~~~
× Failed to compile.
```

How can I integrate?

Thanks.

**Comment by ronaldtse** [comment]
Thanks @devraj112 for reporting this. You will need to also include Opal to run plurimath. 

@hmdne could you please help assist?

**Comment by hmdne** [comment]
This seems like a TypeScript issue, and here unfortunately I don't have much experience.

It seems like definitions from https://github.com/plurimath/plurimath-js/blob/master/src/plurimath-opal.d.ts are not loaded...

**Comment by hmdne** [comment]
@devraj112 I have attempted to fix it in a `plurimath@0.2.1` release. Please reopen this issue if the issue persists.
